ch2.tex

```diff
@@ -40,7 +40,7 @@ \section{Example with some mathematics}
 \addlinespace % a nice non-intrusive separator of data groups (or final table sums)
 Asd qsd 1sd & \textcolor{green!80!black}{GOOD} & 234234299 & -- \\
 Asd & NUMBER & \textbf{123123} & -- \\
-Asd qsd 1sd & DIFFERENT & 234234234 & (no data) \\
+Asd qsd 1sd & \textcolor{orange}{DANGEROUS} & 234234234 & (no data) \\
```

Comment:
This change does not seem to be present in the attached `build/thesis.pdf`.

Reply:
aaaaaaaaand it's in pages.

Extra question, is there any way to make the README link to the built version "dynamic" so that it points to each respective repo's pages?

https://github.com/mff-cuni-cz/better-thesis/pull/5/commits/d2bb52e64000a53abe505c1c70dde8bfefbec4c7

Reply:
(... maybe I could use some magic link to `/raw/` file? )

Reply:
> (I was kinda thinking that the pdf should be built and sent to github pages instead of taking space in the repo forever...might do later today)

I agree that it'd be nicer to build the PDF in GitHub actions and have it published as an artifact. But this is for a separate ticket.

> Extra question, is there any way to make the README link to the built version "dynamic" so that it points to each respective repo's pages?

None that I am aware of.

Reply:
ok let's leave this as is, I'll fix this after someone sends a reasonable complain.

Reply:
Agreed. Thank you.